firmware/CMakeLists.txt

```diff
@@ -71,7 +71,6 @@ else()
         DEPENDS
         brake-upload
         can-gateway-upload
-        null
```

Comment:
This doesn't seem related to the change documented in the commit. I'm approving anyways since it seems like a reasonable change but it is a bit unexpected to see this change too.